sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3425,6 +3425,28 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       assert(SQLConf.get.getConf(SQLConf.CODEGEN_FALLBACK) === true)
     }
   }
+
+  test("Do not display the seed of rand/randn with no argument in output schema") {
```

Comment:
If you don't mind, `SPARK-31594: ` prefix please?

Reply:
Sure!